spec/models/course_spec.rb

```diff
@@ -99,27 +99,33 @@
   end
 
   context 'with site statuses' do
+    let(:findable_site_status) { create(:site_status, :findable) }
+    let(:site_status_with_any_vacancy) { create(:site_status, :with_any_vacancy, course: course) }
+    let(:default_site_status) { create(:site_status, course: course) }
+    let(:applications_being_accepted_now_site_status) { create(:site_status, :applications_being_accepted_now, course: course) }
+    let(:applications_being_accepted_in_future_site_status) { create(:site_status, :applications_being_accepted_in_future, course: course) }
+    let(:site_status_with_no_vacancies) { create(:site_status, :with_no_vacancies, course: course) }
+    let(:course) { findable_site_status.course }
```

Comment:
I know I originally started the pattern of creating a `site_status` and use the course it creates, but I'm not sure this is a great idea, and even it it was then, it's probably not a great idea here, unfortunately. Reason being, I think that we should communicate to the reader that the course is central to the test. Can we create the course in it's own `let` and add it to `findable_site_status` just like we've done with the other `site_statuses`?

I think there'll be an added benefit below, too.